module3/homework/vectorOfSharedPointers/vectorFunctions.cpp

```diff
@@ -0,0 +1,44 @@
+#include "vectorFunctions.hpp"
+
+#include <iostream>
+
+std::vector<std::shared_ptr<int>> generate(int count){
+    std::vector<std::shared_ptr<int>> vec;
+    for(size_t i = 0; i < count; ++i){
+        vec.emplace_back(std::make_shared<int>(i));
+    }
+    return vec;
+}
+void print(std::vector<std::shared_ptr<int>> vec){
+    for(auto el : vec){
+        std::cout << *el << "\n";
+    }
+
```

Comment:
you don't modify vector so 
```suggestion
void print(const std::vector<std::shared_ptr<int>>& vec){
    for(const auto& el : vec){
        std::cout << *el << "\n";
    }

```